package.json

```diff
@@ -24,7 +24,7 @@
     "build": "node-gyp build",
     "readme": "jsdoc2md --template docs/README.hbs index.js > README.md",
     "lint-js": "eslint *.js",
-    "lint-cpp": "cpplint src/*",
+    "lint-cpp": "cpplint --recursive src/*",
```

Comment:
Maybe its cpplint specific? I've used wildcards on Windows on npm scripts before, but this tool seems to take them literally for some reason.